index.js

```diff
@@ -31,9 +32,18 @@ module.exports = function (headerText, data) {
       filename = '';
     }
 
-    var template = data === false ? headerText : gutil.template(headerText, extend({file : file, filename: filename}, data));
+    var template = data === false ? headerText : gutil.template(headerText, extend({ file: file, filename: filename }, data));
     concat = new Concat(true, filename);
 
+    if (fs.lstatSync(file.path).isDirectory()) {
```

Comment:
This broke our build :cry: 
b/c it's stating a file that does not exist and there is no try/catch around this check.